src/CampaignAdmin.php

```diff
@@ -250,6 +250,7 @@ protected function getChangeSetItemResource(ChangeSetItem $changeSetItem)
         $hal = [
             '_links' => [
```

Comment:
I would like to investigate removing the underscore prefix in this pull request, it'd be good to fix this before getting merged